<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SymmetricClichesForm</class>
 <widget class="QWidget" name="SymmetricClichesForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>219</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Clichês Simétricos - Configuração</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>12</y>
     <width>227</width>
     <height>102</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="labelPrimeiraNota">
        <property name="text">
         <string>Primeira Nota</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelLimiteSuperior">
        <property name="text">
         <string>Limite Superior</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelLimiteInferior">
        <property name="text">
         <string>Limite Inferior</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelNumeroCiclos">
        <property name="text">
         <string>Número de Ciclos</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="lineEditFirstPitch"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditLimiteSuperior"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditLimiteInferior"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditNumeroCiclos">
        <property name="inputMask">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>531</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="labelPadraoIntervalar">
      <property name="text">
       <string>Padrão Intervalar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditPadraoIntervalar"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>531</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="3,2">
    <item>
     <widget class="QLabel" name="labelInfo">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="pushButtonOK">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
